test/CPUActivityAgentTest.cpp

```diff
@@ -50,18 +50,11 @@ class CPUActivityAgentTest : public ::testing::Test
             CPU_UNCORE_MAX_CONTROL_IDX
         };
         enum policy_idx_e {
-            CPU_FREQ_MAX = 0,
-            CPU_FREQ_EFFICIENT = 1,
-            CPU_UNCORE_FREQ_MAX = 2,
-            CPU_UNCORE_FREQ_EFFICIENT = 3,
-            PHI = 4,
-            UNCORE_FREQ_0 = 5,
-            UNCORE_MEM_BW_0 = 6,
-            UNCORE_FREQ_1 = 7,
-            UNCORE_MEM_BW_1 = 8,
+            PHI = 0
         };
 
         void SetUp();
+        void SetUpDefaultExpectCalls();
```

Comment:
Despite GTest's naming convention, we still follow our naming convention in test fixture classes (lowercase with underscores).